### PR TITLE
Move Simorgh pages from wiki to docs

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #NUMBER

**Overall change:** 
Move the Simorgh-Pages from wiki -> docs

**Code changes:**
- Add a script which generates the table for all services & environments
- Add the Simorgh-Pages.md

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by amywalkerdev** [comment]
I've been looking into pulling in the urls from the cypress config, as it means less places to update a bunch of urls. Although with just that we miss the launch date info..

**Review by pharingee — APPROVED** [review]
Looks good to me, but maybe we can consider moving the script into it's own file so you don't have to scroll all the way to make changes to it

**Review by ryanmccombe — APPROVED** [review]

**Comment by karinathomasbbc** [comment]
> I've been looking into pulling in the urls from the cypress config, as it means less places to update a bunch of urls. Although with just that we miss the launch date info..

Good shout - I am now reading the config from the cypress config file, and have a separate file in which we can update the launch date.  Hopefully this will make the maintenance of this page much easier.

**Comment by karinathomasbbc** [comment]
I think I will need a way of ignoring these JS files from the coverage checks?

**Comment by ryanmccombe** [comment]
> I think I will need a way of ignoring these JS files from the coverage checks?

There's a .codeclimate.yml file in the repo root that configures CC - can tell it to ignore files in there :)

File: cypress/support/config/allServices.js [hunk 1]
```diff
@@ -0,0 +1,1552 @@
+const isLive = appEnv => appEnv === 'live';
+const isTest = appEnv => appEnv === 'test' || appEnv === 'stage';
+const isLocal = appEnv => appEnv === 'local';
+
+const allServices = appEnv => ({
```
Comment (amywalkerdev) [inline cypress/support/config/allServices.js]: You'll need to update everywhere we import the services in the Cypress tests if renaming this to allServices

Comment (karinathomasbbc) [reply cypress/support/config/allServices.js]: The only reference is in `cypress/support/config/services.js` but I will double check all of the Cypress tests are running & passing OK before pushing

**Pull request merged**